### PR TITLE
case-app migration of cmta - compiling and tests passing

### PR DESCRIPTION
**Pull request by thinkmorestupidless**

### CODE REVIEW

File: cmta/src/main/scala/cmt/admin/command/Studentify.scala [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+        } yield result
+      else
+        sbtio.delete(tmpFolder)
+        Right(successMessage)
+    end buildStudentifiedRepository
+  end StudentifyHelpers
+
+  val command = new CmtCommand[Studentify.Options] {
+
+    def run(options: Studentify.Options, args: RemainingArgs): Unit =
```
Comment (eloots) [inline cmta/src/main/scala/cmt/admin/command/Studentify.scala:144]: Wondering what the `args` parameter is doing as it doesn't seem to be utilised in the `run` method.

**Review by eloots — COMMENTED** [review]
There seems to be an issue with setting the mainRepository name. This seems to be true for all `cmta` commands.

**Review by eloots — APPROVED** [review]
A great improvement over `scopt`! Much simpler...

File: core/src/main/scala/cmt/Error.scala [hunk 2]
```diff
@@ -58,19 +60,4 @@ extension (self: caseapp.core.Error)
...
-           |${reasons.mkString("   ", "\n   ", "\n")}""".stripMargin
-      case FailedToExecuteCommand(reason) =>
-        s"""Failed to execute command:
-           |  ${toConsoleYellow(reason.message)}
-           |""".stripMargin
-      case FailedToValidateCommandOptions(reasons) =>
-        s"""To be implemented:
-           |${reasons}""".stripMargin
-    }
+extension (cmtError: CmtError) def prettyPrint: String = cmtError.toDisplayString
```
Comment (thinkmorestupidless) [inline core/src/main/scala/cmt/Error.scala:63]: so `prettyPrint` is, effectively, just an alias for `toDisplayString`?

should we just change the name of `toDisplayString` to `prettyPrint`?

**Pull request merged**